core/src/main/java/com/datastrato/gravitino/meta/BaseMetalake.java

```diff
@@ -24,6 +24,8 @@
 @ToString
 public class BaseMetalake implements Metalake, Entity, Auditable, HasIdentifier {
 
+  public static final String SYSTEM_METALAKE_RESERVED_NAME = "system";
```

Comment:
We'd better have a place to manage all the reserved names, not split them all around.

Reply:
Done.